WooCommerce/Classes/Model/ShippingLabelPackageAttributes.swift

```diff
@@ -16,3 +16,30 @@ struct ShippingLabelPackageAttributes: Equatable {
     /// List of items in the package.
     let items: [ShippingLabelPackageItem]
 }
+
+extension ShippingLabelPackageAttributes {
+    var isOriginalPackaging: Bool {
+        packageID == Self.originalPackagingBoxID
+    }
+
+    func partitionItems(using productOrVariationID: Int64) -> (matchingItem: ShippingLabelPackageItem?, rest: [ShippingLabelPackageItem]) {
```

Comment:
minor nit: I think a small description would be useful to understand what it does and where it will be used, without necessarily having to read the code.

Reply:
I have updated the comment in f29683e1e7e6031c34cb75075987e9b0328f084e - thanks for the suggestion!